ingest-app/src/cmr/ingest/services/bulk_update_service.clj

```diff
@@ -165,34 +177,45 @@
         ;; Write db rows - one for overall status, one for each concept id
         task-id (try
                   (data-bulk-update/create-bulk-update-task
-                    context provider-id json concept-ids)
-                 (catch Exception e
-                   (let [msg (.getMessage e)
-                         msg (if (string/includes? msg "BULK_UPDATE_TASK_STATUS_UK")
-                               "Bulk update name needs to be unique within the provider."
-                               msg)]
-                     (errors/throw-service-errors
+                   context provider-id json concept-ids)
+                  (catch Exception e
+                    (let [msg (.getMessage e)
+                          msg (if (string/includes? msg "BULK_UPDATE_TASK_STATUS_UK")
+                                "Bulk update name needs to be unique within the provider."
+                                msg)]
+                      (errors/throw-service-errors
                        :invalid-data
                        [(str "Error creating bulk update task: " msg)]))))]
     ;; Queue the bulk update event
     (ingest-events/publish-ingest-event
-      context
-      (ingest-events/ingest-bulk-update-event provider-id task-id bulk-update-params user-id))
+     context
+     (ingest-events/ingest-bulk-update-event provider-id task-id bulk-update-params user-id))
     task-id))
 
+(defn validate-and-save-bulk-granule-update
```

Comment:
I was working on breaking the granule bulk update logic into its own file. Is there enough common functionality here that it's worth having everything in the same `_service.clj` file?

Reply:
I'm also interested in the answer to this

Reply:
This code will be merged as is. The splitting will take place in follow-on code to prevent merge conflicts.